components/HabitList.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+import { HABITS } from "../lib/db";
+
+function Card({ habit }) {
+  return (<ul>{habit.id}</ul>), (<p>{habit.name}</p>);
+}
+export function HabitList() {
+  return (
+    <Main>
+      <h3>All Day</h3>
+      <ul>
+        {HABITS.map((habitElement) => {
+          return <Card key={habitElement.id} habit={habitElement} />;
+        })}
+      </ul>
+    </Main>
```

Comment:
I guess you should refactor this part, the HabitList should only be the ul list, Main and h3 can be put into the index.js

Reply:
just to make sure. the HabitList is not required I can move  

   ``` <h3>All Day</h3>
       <ul>
         {HABITS.map((habitElement) => {
           return <Card key={habitElement.id} habit={habitElement} />;
         })}
       </ul> 
```


to index.js

Reply:
> I guess you should refactor this part, the HabitList should only be the ul list, Main and h3 can be put into the index.js


just to make sure. the HabitList is not required I can move  

   ``` <h3>All Day</h3>
       <ul>
         {HABITS.map((habitElement) => {
           return <Card key={habitElement.id} habit={habitElement} />;
         })}
       </ul> 
```


to index.js